app/assets/stylesheets/custom.scss

```diff
@@ -321,3 +321,25 @@ ul, menu, dir {
     width: 100px;
     height: 30px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
+          background-color: #FF6565;
+        }
+  &.notice {
+    border: solid 4px #26722D;
+          background-color: #A4E7A0;
+        }
+  color: black;
+  z-index: 999;
+  position: fixed;
+  font-weight: bold;
+  width: 96%;
+  margin: 20px;
+  padding-top: 50px;
+        p {
```

Comment:
Rule declaration should be preceded by an empty line<br>Line should be indented 2 spaces, but was indented 8 spaces